content/_index.md

```diff
@@ -1,99 +1,173 @@
+
+**Wir schaffen Tech-Projekte mit gesellschaftlichem Mehrwert.** Wir
+programmieren Anwendungen, die Menschen helfen. Umweltsensoren zum
+Selberbauen und eine Trinkwasser-App sind nur zwei unserer Projekte.
+All unser Code ist frei verwendbar und Open Source.
+
+**Wir unterstützen Städte und Kommunen bei der Digitalisierung.**
+Mit Beratung, Wissensvermittlung und durch unsere kostenfrei
+weiterverwendbaren Open-Source-Projekte helfen wir
+Verwaltungsmitarbeiter\*innen und Politiker\*innen.
+
+**Wir leisten Überzeugungsarbeit.** Auf Vorträgen, in Publikationen
+und in Diskussionen rund um Technik und Digitalisierung bringen wir
+die Sichtweise der Zivilgesellschaft ein. Wir analysieren aktuelle
+Geschehnisse auf unserem Blog. Unsere Forderungen dienen dem Wohl
+der Gemeinschaft. Damit sind wir ein wichtiges Gegenstück zu
+Akteur\*innen mit wirtschaftlichen Interessen.
+
+**Wir schaffen eine Community.** Die Welt verbessern funktioniert
+leichter zusammen. Bei uns versammeln sich Menschen aus der
+Hacker\*innen-Szene, dem digitalen Ehrenamt, der engagierten
+Zivilgesellschaft, aus Verwaltung und Politik. Wir inspirieren
+einander, arbeiten zusammen und lernen voneinander.
+
+**Wir stecken in vielen Netzwerken.** Im globalen Netzwerk von <a href="https://codeforall.org/">Code
+for All</a> sind wir mit Civic-Tech-Organisationen auf der ganzen Welt
+verbunden. Durch unseren Trägerverein [Open Knowledge Foundation
+Deutschland](https://okfn.de) sind wir Teil des internationalen Open Knowledge
+Netzwerks.
+
+{{< headline-brackets-h2 color="blue" >}}
+Informationen für Verbündete und solche, die es werden wollen
+{{</ headline-brackets-h2 >}}
+
+### Verwaltungen
+
+Verwaltungen müssen die nötige IT-Expertise in ihren eigenen
+Abteilungen fördern, anstatt Digitalisierungsprojekte an externe
+Dienstleister\*innen auszulagern. Sie müssen einheitliche Datenstandards
+durchsetzen, Offene Daten zur Regel machen und Open Source Software
+einsetzen. Damit handeln sie selbstbestimmt, werden unabhängig von
+Konzernen, arbeiten nachhaltig und gemeinwohlorientiert.
+
+Mit unserer Expertise unterstützen wir Menschen in der Verwaltung
+auf ihrem Weg zu einer nachhaltigen Digitalisierung. Interessierte
+Verwaltungsmitarbeiter\*innen informieren sich in der [Open Data
+Knowledgebase](https://opendata.okfn.de/), kontaktieren ihre 
+[lokale Community](/community) oder schreiben
+uns eine [Nachricht](mailto:info@codefor.de?subject=Kontakt).
+
+Gerne teilen wir unser Wissen im Dialog auf Augenhöhe, als Ehrenamtliche
+meist außerhalb regulärer Bürozeiten. Wie gute Zusammenarbeit
+gelingt, beschreibt der Ratgeber [Digitales Ehrenamt im Kontext der
+Kommunalen
+Verwaltung](https://library.fes.de/pdf-files/bueros/stuttgart/20087-20230313.pdf),
+verfasst von zwei Ehrenamtlichen unserer Community.
+
+<div class="landing-callout">
+In der Blogserie <a href="/blog">Out in the Open</a> analysieren unsere
+Expert\*innen aktuelle Ereignisse zu Digitalpolitik, Open Data und
```

Comment:
```suggestion
Expert\*innen aktuelle Ereignisse zu Digitalpolitik, Open Data und
```

Reply:
keine ahnung warum das hier mit dem Sternchchen \* nicht funktioniert? vllt hatte ich wieder falsches Zeichen, das gleich aussah...

Reply:
Das lag daran, dass das in einem `<div class="landing-callout">` Container lag und dann da das Markdown Processing abgeschaltet wurde und stattdessen nur html gerendert. Damit das in Zukunft nicht verwirrt habe ich das `landing-callout` in einen Hugo Shortcode verwendet, da kann man dann ganz normalen Markdown Content schreiben wie sonst überall auch :)

Reply:
thank you for this extra work.